modules/doc/content/getting_started/installation/install_mfem.md

```diff
@@ -0,0 +1,64 @@
+# Installing MFEM-MOOSE
+
+To enable `MFEM-MOOSE` capabilities, it is necessary to install all of its dependencies, including `MFEM` itself and `Conduit`. You may do so by performing the following steps:
+
+1. After cloning the `MOOSE` repository, navigate to the root directory and run the following commands:
+
+```bash
+export METHOD=opt
+export MOOSE_JOBS=10
+```
+
+First, we need to build `PETSc`. To do that, we can run the `PETSc` installer script:
+
+```bash
+./scripts/update_and_rebuild_petsc.sh [options]
+```
+
+You may substitute `[options]` by any `PETSc` configure flags, which are discussed in the [`PETSc` install page](https://petsc.org/release/install/install). If you intend to build `MFEM-MOOSE` with GPU offloading capabilities, make sure to include `--with-cuda` or `--with-hip` to the flags. If desired, you may also specify the GPU architecture with `--with-cuda-arch=[arch]` or `--with-hip-arch=[arch]`.
```

Comment:
I have a vague recollection that there are warnings, maybe just from PETSc tests themselves, if you use `--with-cuda` in PETSc if your mpi has not been configured with cuda support. We might want to mention something about ideally having a gpu-aware mpi installation